src/pages/nodejs/file-system/index.md

```diff
@@ -1,13 +1,82 @@
----
-title: File System
----
 ## File System
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/nodejs/file-system/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+The Node.js File System module allows you to work with the file system on your computer.
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+What is a file system module ?
```

Comment:
Remove this line. Since the scope the document is on File System, I do not think there is any need for duplication.